tck/pom.xml

```diff
@@ -27,6 +27,135 @@
             <version>6.9.9</version>
             <scope>compile</scope>
         </dependency>
+        
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
I didn't include anything impl specific - my implementation resides in a completely different repo, which only depends on the TCK maven module.
The additional dependencies are only to support testing (JUnit, Hamcrest) or make Arquillian work out of the box (Weld connector). 

If the point is to remove the dependency to the Weld connector and force the implementation to pull an appropriate connector, I agree we can go this way.

If the point is to choose Test-NG as the single and unified testing framework, I don't mind. I chose jUnit just as a temporary solution because I'm more productive with it. I plan to rewrite the tests to TestNG, I just wanted to focus on the proposed API.